packages/core/src/modules/vc/W3cCredentialApi.ts

```diff
@@ -0,0 +1,46 @@
+import type { W3cVerifiableCredential } from './models'
+import type { StoreCredentialOptions } from './models/W3cCredentialServiceOptions'
+import type { W3cCredentialRecord } from './repository'
+import type { Query } from '../../storage/StorageService'
+
+import { AgentContext } from '../../agent'
+import { injectable } from '../../plugins'
+
+import { W3cCredentialService } from './W3cCredentialService'
+
+@injectable()
```

Comment:
do you think some light tests can be added for the public api? Maybe just one test that stores a credential, fetches it by id, fetches it using the find by id, deletes it and then expects getAllRecords to return an empty array or something.

Reply:
Done!